modules/tinymce/CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed the deprecated `filepicker_validator_handler`, `force_p_newlines`, `gecko_spellcheck`, `tab_focus`, `table_responsive_width` and `toolbar_drawer` settings #TINY-7820
 - Removed the deprecated `editor_deselector`, `editor_selector`, `elements`, `mode` and `types` legacy TinyMCE init settings #TINY-7822
 - The legacy `mobile` theme has been removed #TINY-7832
+- Removed Internet Explorer 11 from the automated test suite #TINY-8194
```

Comment:
```suggestion
- Removed support for Microsoft Internet Explorer 11 #TINY-8194
```
The fact it was removed from tests is not a customer facing change. So we should either include something like this or just not include a changelog entry.